test/test.js

```diff
@@ -1278,7 +1326,7 @@ describe("follow-redirects", function () {
 
       var opts = url.parse("http://127.0.0.1:3600/a");
       opts.headers = {
-        host: "localhost",
+        host: "localhost:3600",
```

Comment:
We probably also want a negative test here, when we redirect to `localhost` (no port) that it drops the `Authorization` header, which it incorrectly does not do now.